cmd/agent/app/reporter/metrics.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package reporter
+
+import (
+	"github.com/uber/jaeger-lib/metrics"
+
+	"github.com/jaegertracing/jaeger/thrift-gen/jaeger"
+	"github.com/jaegertracing/jaeger/thrift-gen/zipkincore"
+)
+
+const (
+	jaegerBatches = "jaeger"
+	zipkinBatches = "zipkin"
```

Comment:
see comments in the Process deduping PR, maybe we don't need this separation anymore (we will already have metrics from udp zipkin/jaeger thrift servers)

Reply:
do you mean if we kill the zipkin thrift in the agent?

Reply:
in the agent's reporter, e.g. https://github.com/jaegertracing/jaeger/pull/1181#discussion_r234053131